modules/metering-install-verify.adoc

```diff
@@ -0,0 +1,66 @@
+// Module included in the following assemblies:
+//
+// * metering/metering-installing-metering.adc
+
+[id="metering-install-verify_{context}"] 
+= Verify Installation 
+
+To verify your metering installation you can check that all the required pods have been activated, and check that your report data sources are beginning to import data.
+
+.Procedure
+
+. Navigate to *Workloads* -> *Pods* in the metering namespace and verify that pods are being created. This can take up to a minute after installing the metering stack.
+
+You can run the same check using kubectl:
+
+----
+kubectl -n openshift-metering get pods
```

Comment:
Throughout the PR - recommend changing these commands to use "oc" instead of "kubectl". And if so, changing the lead-in sentence to say "using the `oc` CLI".